PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -2,7 +2,7 @@ javaVersion=1.8
 org.gradle.jvmargs='-Duser.language=en'
 sources=false
 intellij_version=IC-2020.1
-dep_plugins=org.intellij.scala:2020.1.25
+dep_plugins=org.intellij.scala:2020.1.36
```

Comment:
Maybe we need to change scala version at other place to keep consistency. Check #4166 for more details.

Reply:
Fixed.